### PR TITLE
Stripes refactor

### PR DESCRIPTION
**Pull request by hea3ven**

### CODE REVIEW

File: api/buildcraft/api/transport/IStripesHandler.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 public interface IStripesHandler {
 	public static enum StripesHandlerType {
 		ITEM_USE,
+		ITEM_USE_LAST,
```
Comment (asiekierka) [inline api/buildcraft/api/transport/IStripesHandler.java]: I don't think having a publicly available ITEM_USE_LAST is a good idea. Consider making StripesHandlerDefault added to the end of the list of all stripes handlers from the handling code.

**Comment by asiekierka** [comment]
The rest looks good to me!

File: api/buildcraft/api/transport/IStripesHandler.java (hunk 1 above)
Comment (hea3ven) [inline api/buildcraft/api/transport/IStripesHandler.java]: Yeah, I agree. The other option I considered is splitting handlers into two lists, high priority handlers and low priority handlers.

Comment (asiekierka) [inline api/buildcraft/api/transport/IStripesHandler.java]: Ping?

**Pull request merged**